crate/README.md

```diff
@@ -62,12 +62,11 @@ the latest run of the test.
 Inside each individual run of a test you can find the log files and results of
 the test run. More specifically you can see:
 
- - ``jepsen.log`` containing logging information of the test run
+ - ``jepsen.log`` containing logging information of the test run.
  - ``timeline.txt`` graphical timeline representation of the events that took 
    place. 
- - ``latency-quantiles.png``
- - ``latency-raw.png``
- - ``rate.png``
+ - ``latency-quantiles.png``, ``latency-raw.png``, ``rate.png`` various graphs
+   related to the events that took place during the test run.
  - ``n[1-5]`` directories each one containing the ``crate.log`` file of each of
```

Comment:
maybe this si a bit more clear "a directory for every crate node configured for the test, each containing the crate log"